README.md

```diff
@@ -4,7 +4,11 @@ This is the product page for GovWifi. It is a [middleman site](http://middlemana
 
 ## Getting Setup for Developement
 
-The recommended ruby version is `2.7.2`. You may need to install this.
+Packages required to be installed to get the local version working are:
```

Comment:
maybe just 'Software' instead of 'Packages' - I think its only called packages in linux

Reply:
Sure